tools/GetGrammar/grammar-lexer-members.txt

```diff
@@ -0,0 +1,10 @@
+
+@lexer::members {
```

Comment:
@RexJaeschke 

This appears to be incorrect syntax. Where should we look for this?

Reply:
OK, it was completely my fault for using `>` instead of `>>`.

This is fixed now.